src/components/MDX/MDXComponents.tsx

```diff
@@ -368,8 +369,28 @@ function YouTubeIframe(props: any) {
   );
 }
 
-function Image(props: any) {
-  return <img className="max-w-[calc(min(700px,100%))]" {...props} />;
+function Image(allProps: any) {
+  const {height, width, style, ...props} = allProps;
+  return (
+    <div
+      style={{
+        ...(!style ? {} : style),
+        ...(!width ? {} : {width: `${width}px`}),
+        ...(!height ? {} : {maxHeight: `${height}px`}),
```

Comment:
actually you could remove the `!` and swap the conditions too :p

Reply:
Done